dashboard/templates/base.html

```diff
@@ -13,9 +13,10 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>{% use_macro title %}</title>
-    <link rel="stylesheet" type="text/css" href="/static/@fortawesome/fontawesome-free/css/all.css">
-    <link rel="stylesheet" type="text/css" href="/static/@fortawesome/fontawesome-free/css/fontawesome.css">
-    <link rel="stylesheet" type="text/css" href="/static/@fortawesome/fontawesome-free/css/fontawesome.min.css">
+    <link rel="stylesheet" type="text/css" href="{% static '@fortawesome/fontawesome-free/css/all.css' %}">
+    <link rel="stylesheet" type="text/css" href="{% static '@fortawesome/fontawesome-free/css/fontawesome.css' %}">
+    <link rel="stylesheet" type="text/css" href="{% static '@fortawesome/fontawesome-free/css/fontawesome.min.css' %}">
+    <link rel="shortcut icon" href="{% static '/images/favicon.ico' %}"/>
```

Comment:
I think the `/` in front of `images` is a bug. When I start up the app, it works initially, but when I click around and log in and out as a regular user, I see a `SuspiciousFileOperation` exception.

Reply:
Yeah it's a bug, SuspiciousFileOperation! Wow!

Reply:
```suggestion
    <link rel="shortcut icon" href="{% static 'images/favicon.ico' %}"/>
```

Reply:
Also <link> doesn't have an end tag in HTML, so fixing that.